octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/FieldEditorFactory.java

```diff
@@ -58,19 +91,27 @@ public FieldEditor createFieldEditor(Composite parent, EntityModelWrapper entity
                 case DateTime:
                     fieldEditor = new DateTimeFieldEditor(parent, SWT.NONE);
                     break;
+                case Reference:
+                    try {
+                        fieldEditor = createReferenceFieldEditor(parent, entityModelWrapper, fieldMetadata);
+                    } catch (Exception e) {
+                        log.log(new Status(IStatus.ERROR, Activator.PLUGIN_ID, "Failed to create reference field editor: " + e));
+                        fieldEditor = new ReadOnlyFieldEditor(parent, SWT.NONE);
+                    }
+                    break;
                 default:
                     fieldEditor = new ReadOnlyFieldEditor(parent, SWT.NONE);
                     break;
             }
-
         }
-        
+
         try {
+
             fieldEditor.setField(entityModelWrapper, fieldName);
+
         } catch (Exception ex) {
-            ILog log = Activator.getDefault().getLog();
             StringBuilder sbMessage = new StringBuilder();
-            sbMessage.append("Faied to set field ")
+            sbMessage.append("Faied to set field  ")

```

Comment:
@atothhpe Why did you add an empty space? And since you touched this can you change to "Failed?

Reply:
accident, fixing